public/video-ui/src/reducers/uploadsReducer.js

```diff
@@ -0,0 +1,21 @@
+export default function upload(state = null, action) {
```

Comment:
This component seems to have two "resting" states... it's `null` before an upload, but `{completed: true}` after an upload. Is that correct? Thinking about state when you upload a file... go out of screen then back in to upload another without refreshing